SimPEG/EM/TDEM/SrcTDEM.py

```diff
@@ -46,11 +46,16 @@ def evalDeriv(self, time):
 
 class StepOffWaveform(BaseWaveform):
 
+    eps = 1e-9
+
     def __init__(self, offTime=0.):
         BaseWaveform.__init__(self, offTime=offTime, hasInitialFields=True)
 
     def eval(self, time):
-        return 0.
+        if abs(time-0.) < self.eps:
```

Comment:
We do need that. It is not only just an initial field, but also with the Se term. When solving either b or h no problem, but when solving we do need time derivative of Se on the rhs indicating we need to take of previous time of Se.